---
layout: about
title: about
permalink: /
subtitle: <a href='#'></a>

profile:
  align: right
  image: prof_pic.jpg
  image_circular: true # crops the image to make it circular
  address: >
    <p>Computational biologist</p>
    <p>Postdoctoral fellow at Genome Sciences department, UW</p>

news: false  # includes a list of news items
latest_posts: false  # includes a list of the newest posts
selected_papers: false # includes a list of papers marked as "selected={true}"
social: false  # includes social icons at the bottom of the page
---

[Google Scholar](https://scholar.google.com/citations?user=mnTncLgAAAAJ&hl=en/), 
[ORCID](https://orcid.org/0000-0002-7630-1251)

I am a UW Data Science postdoctoral fellow with [Dr. William Noble](https://noble.gs.washington.edu/~wnoble/) at the University of Washington. 
Before this, I did my undergraduate studies at Tsinghua University, and my graduate studies at Princeton University, supervised by [Dr. Olga Troyanskaya](https://function.princeton.edu/).

I am interested in developing machine learning methods to tackle biology-related problems. 
I started as a wet-lab biologist in undergrad, where I used a combination of experimental approaches -- including imaging, biochemistry, cell biology -- to study autophagy in human cell lines and yeast.
My graduate work is focused on integrating large-scale functional genomics datasets and applying network-based approaches to predict context-specific disease genes and pathways in neurodevelopmental and neurodegenerative diseases and disorders.
During my postdoc studies, I develop deep learning methods to integrate and translate single-cell multi-omics profiles across data modality, time and species, to generate biological hypothesis in biological contexts or data modalities where experimental measurement is scarse.

My research interests include:
* Single-cell cross-modality translation to predict cellular profiles in missing data modalities
* Cross-species imputation to transfer knowledge from model organisms to human
* Reprioritization of disease genes using context-specific gene-gene relationship networks
* Data integration across modalities, biological contexts, time, and species
